tests/foreman/ui/test_host.py

```diff
@@ -513,6 +519,128 @@ def test_positive_delete(self):
+        be present in the list
+
+        @id: a4affb90-1222-4d9a-94be-213f9e5be573
+
+        @expectedresults: All assigned hosts to organization are returned by
+        search
+
+        @CaseLevel: Integration
+        """
+        org = entities.Organization().create()
+        param_name = gen_string('alpha')
+        param_value = gen_string('alpha')
+        parameters = [{'name': param_name, 'value': param_value}]
+        param_host = entities.Host(
+            organization=org,
+            host_parameters_attributes=parameters,
+        ).create()
+        additional_host = entities.Host(organization=org).create()
+        with Session(self.browser) as session:
+            set_context(session, org=org.name)
+            # Check that hosts present in the system
+            for host in [param_host, additional_host]:
+                self.assertIsNotNone(self.hosts.search(host.name))
+            # Check that search by parameter with 'not' prefix returns both
+            # hosts in the list
+            self.assertIsNotNone(
+                self.hosts.search(
+                    param_host.name,
+                    _raw_query='not params.{0} = {1}'.format(
+                        param_name, gen_string('alphanumeric'))
+                )
+            )
+            strategy, value = locators['host.select_name']
+            self.assertIsNotNone(self.hosts.wait_until_element(
+                (strategy, value % additional_host.name)))
+
+    @run_only_on('sat')
+    @skip_if_bug_open('bugzilla', 1392422)
+    @tier2
+    def test_positive_search_by_parameter_with_operator(self):
+        """Search by global parameter assigned to host using operator '<>' and
+        any random string as parameter value to make sure that all hosts will
+        be present in the list
+
+        @id: 264065b7-0d04-467d-887a-0aba0d871b7c
+
+        @expectedresults: All assigned hosts to organization are returned by
+        search
+
+        @BZ: 1392422
+
+        @CaseLevel: Integration
+        """
+        org = entities.Organization().create()
+        param_name = gen_string('alpha')
+        param_value = gen_string('alpha')
+        parameters = [{'name': param_name, 'value': param_value}]
+        param_host = entities.Host(
+            organization=org,
+            host_parameters_attributes=parameters,
+        ).create()
+        additional_host = entities.Host(organization=org).create()
+        with Session(self.browser) as session:
+            set_context(session, org=org.name)
+            # Check that hosts present in the system
+            for host in [param_host, additional_host]:
+                self.assertIsNotNone(self.hosts.search(host.name))
+            # Check that search by parameter with '<>' operator returns both
+            # hosts in the list
+            self.assertIsNotNone(
+                self.hosts.search(
+                    param_host.name,
+                    _raw_query='params.{0} <> {1}'.format(
```

Comment:
yes, I spent like two hours and tried everything I could, but Selenium just sent '=' instead '!=' all the time. Anyway, I will try few more approaches